pydantic_ai_slim/pydantic_ai/models/function.py

```diff
@@ -239,7 +239,8 @@ def _estimate_usage(messages: Iterable[ModelMessage]) -> usage.Usage:
         if isinstance(message, ModelRequest):
             for part in message.parts:
                 if isinstance(part, (SystemPromptPart, UserPromptPart)):
-                    request_tokens += _estimate_string_tokens(part.content)
+                    # TODO(Marcelo): I'll support this.
```

Comment:
don't know what this means

Reply:
That I still need to implement before merging it.